diagnostic.md

```diff
@@ -32,23 +35,30 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+getters:
+index()
+get()
```

Comment:
No. Examples of getters are `.html()`, and `.css('<some property>')`. Examples of setters are `.html('some html')` and `.css('<some property>, <some value>')`.